### PR TITLE
MAINT: add threads/jobs primitives

### PR DESCRIPTION
**Pull request by colinvwood**

### CODE REVIEW

File: q2_feature_classifier/_blast.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
     FeatureData, Taxonomy, Sequence, DNAFASTAFormat, DNAIterator, BLAST6,
     BLAST6Format)
 from .types import BLASTDBDirFmtV5, BLASTDB
-from qiime2.plugin import Int, Str, Float, Choices, Range, Bool
+from qiime2.plugin import Int, Str, Float, Choices, Range, Bool, Threads
+from qiime2.sdk.util import get_available_cores
```
Comment (lizgehret) [inline q2_feature_classifier/_blast.py]:
```suggestion
```

File: q2_feature_classifier/_blast.py [hunk 2]
```diff
@@ -14,7 +14,8 @@
     FeatureData, Taxonomy, Sequence, DNAFASTAFormat, DNAIterator, BLAST6,
     BLAST6Format)
 from .types import BLASTDBDirFmtV5, BLASTDB
-from qiime2.plugin import Int, Str, Float, Choices, Range, Bool
+from qiime2.plugin import Int, Str, Float, Choices, Range, Bool, Threads
```
Comment (lizgehret) [inline q2_feature_classifier/_blast.py]:
```suggestion
from qiime2.plugin import (Int, Str, Float, Choices, Range, Bool, Threads,
                                             get_available_cores)
```

File: q2_feature_classifier/classifier.py [hunk 3]
```diff
@@ -14,10 +14,11 @@
 import subprocess
 
 import pandas as pd
-from qiime2.plugin import Int, Str, Float, Bool, Choices, Range
+from qiime2.plugin import Int, Str, Float, Bool, Choices, Range, Threads
```
Comment (lizgehret) [inline q2_feature_classifier/classifier.py]:
```suggestion
from qiime2.plugin import (Int, Str, Float, Bool, Choices, Range, Threads,
                                             get_available_cores)
```

File: q2_feature_classifier/classifier.py [hunk 4]
```diff
@@ -14,10 +14,11 @@
...
 from q2_types.feature_data import (
     FeatureData, Taxonomy, Sequence, DNAIterator, DNAFASTAFormat)
 from q2_types.feature_table import FeatureTable, RelativeFrequency
+from qiime2.sdk.util import get_available_cores
```
Comment (lizgehret) [inline q2_feature_classifier/classifier.py]:
```suggestion
```

**Review by lizgehret — APPROVED** [review]

**Comment by colinvwood** [comment]
blocked by qiime2/qiime2#750

**Review by lizgehret — APPROVED** [review]

**Pull request merged**